src/components/01-branding/02-typography/callouts/large-alerts.njk

```diff
@@ -0,0 +1,42 @@
+<table class="usa-table">
+  <caption>Large Alerts</caption>
+  <thead>
+    <tr>
+      <th scope="col">Heading</th>
+      <th scope="col">Example</th>
+      <th scope="col">Configuration</th>
+    </tr>
+  </thead>
+  <tbody>
```

Comment:
Global and page are identical but please update to use BEM classes if distinction needs to be made